C_CPP/Stack/readme.md

```diff
@@ -1 +1,7 @@
-
+-
+-STACK(NEAREST GREATER TO RIGHT IN STACK)
+-
+-
+-Stack foundation problem(earest greater to right or say Next greater element 
```

Comment:
after each line of explanation, pls leave one line space , so that when u preview the .md file it, it comes in the next line as bullet points

Reply:
ok

Reply:
can you please check now.

Reply:
can you please check the PR now , i have resolved the errors.